understanding/use-of-color.html

```diff
@@ -55,294 +53,206 @@
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#resources">関連リソース</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 1.4.1:</span>Use of Color (Level A)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 1.4.1:</span>色の使用 (レベル A)
+
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>Color is not used as the only visual means of conveying information, indicating an
-                     action, prompting a response, or distinguishing a visual element.
+                  <p>色が、情報を伝える、動作を示す、反応を促す、又は視覚的な要素を判別するための唯一の視覚的手段になっていない。
                      
                   </p>
                   <div class="note">
-                     <p class="note-title marker">Note</p>
-                     <p>This success criterion addresses color perception specifically. Other forms of perception
-                        are covered in <a href="#adaptable">Guideline 1.3</a> including programmatic access to color and other visual presentation coding.
+                     <p class="note-title marker">注記</p>
+                     <p>この達成基準は、特に色の知覚に関するものである。その他の知覚形態については、色やその他の視覚的提示のコーディングへのプログラムによるアクセスも含めて、<a href="adaptable.html">ガイドライン 1.3</a> で網羅されている。
+
                      </p>
                   </div>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Color is not the only way of distinguishing information.</dd>
+                  <dt>目標</dt>
+                  <dd>色だけが、情報を判別する唯一の方法とならないようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Use information in addition to color, such as shape or text, to convey meaning.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>色だけでなく、形状やテキストといった情報も用いて、意味を伝達する。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>Not everyone sees colors or sees them the same way.</dd> 
+                  <dt>なぜそれが重要か</dt>
+                  <dd>誰もが色を見たり、又は色を同じように見たりできるわけではない。</dd> 
```

Comment:
「が」が多いのが微妙ですが、こうでしょうか…？
```suggestion
                  <dd>誰もが色が見えるわけでも、色が同じように見えるわけでもない。</dd> 
```

Reply:
@momdo 「〜なわけでも、〜なわけでもない。」という表現は適切だと思います。ただやはり「が」が多くて若干読みにくい感じはありますね。「everyone」を直訳的に「全ての人」にすると、わかりやすい感じになりますかね...？

> 全ての人が色が見えるわけでも、色が同じように見えるわけでもない。

(さらにこねくり回して)
> 全ての人にとって、色が見えるわけでも、色が同じように見えるわけでもない。

Reply:
@caztcha 

「全ての人」はよいですね…！

Reply:
ありがとうございます。[18d6198](https://github.com/waic/wcag22/pull/330/commits/18d61989e195b4b01cc5644691e3bb3f00b4bc1a) にて修正しました。